src/commands/Start.ts

```diff
@@ -20,22 +18,21 @@ function error(err: string): void {
 async function run(): Promise<void> {
   try {
     log.info('Starting Enveloping Relay Server process...\n');
-    const contractsConfig: ContractsConfig = config.get('contracts');
-    const appConfig: AppConfig = config.get('app');
-    log.setLevel(appConfig.logLevel);
-    if (!config.has('blockchain.rskNodeUrl')) {
+    const { contracts, app, blockchain } = getServerConfig();
+    log.setLevel(app.logLevel);
+    if (!blockchain.rskNodeUrl) {
       error('missing rskNodeUrl');
     }
-    const trustedVerifiers = contractsConfig.trustedVerifiers;
+    const trustedVerifiers = contracts.trustedVerifiers;
 
     log.debug('runServer() - provider done');
     // config = await resolveServerConfig(conf, provider);
     log.debug('runServer() - config done');
     if (trustedVerifiers && trustedVerifiers.length > 0) {
-      contractsConfig.trustedVerifiers = trustedVerifiers;
+      contracts.trustedVerifiers = trustedVerifiers;
     }
```

Comment:
What's the purpose of this? `trustedVerifiers` is already taken from the `contracts` object, why do we re-assign it?

Reply:
It looks like it's not necessary that part any more.

Reply:
updated